lib/bandiera/web_app_base.rb

```diff
@@ -11,7 +11,9 @@ class InvalidParams < StandardError; end
       enable :logging
       enable :raise_errors if ENV['AIRBRAKE_API_KEY'] && ENV['AIRBRAKE_PROJECT_ID']
 
-      set :feature_service, CachingFeatureService.new(FeatureService.new)
+      set :feature_service, CachingFeatureService.new(FeatureService.new,
+        cache_size: ENV['CACHE_SIZE']&.to_i,
+        cache_ttl: ENV['CACHE_TTL']&.to_i)
```

Comment:
This will set all other users cache size and ttl to `nil` unless they explicitly set the ENV vars (meaning they won't have a TTL at all and cache entries will be permanent).

```
10:21 $ pry
[1] pry(main)> nil.to_i
=> 0
[2] pry(main)> ENV['CACHE_SIZE']&.to_i
=> nil
[3] pry(main)> def test(cache_size: 100)
[3] pry(main)*   puts cache_size.inspect
[3] pry(main)* end
=> :test
[4] pry(main)> test(cache_size: ENV['CACHE_SIZE']&.to_i)
nil
=> nil
```

We probably want to put this in the registry pattern in `lib/bandiera.rb` so we can set the defaults there.  Want me to quickly submit a change?

Reply:
That's true, which is why the initialiser is nil-safe:
```
@cache = LruRedux::TTL::ThreadSafeCache.new(cache_size || 100, cache_ttl || 10)
```

> We probably want to put this in the registry pattern in lib/bandiera.rb so we can set the defaults there. Want me to quickly submit a change?

Please, any enhancements are very welcome 😄

Reply:
I've just pushed a change to move the default values to constants, which hopefully makes what's happening here a little more obvious.